cms/static/coffee/spec/main.coffee

```diff
@@ -180,10 +178,13 @@ requirejs.config({
             exports: "sinon"
         },
         "jasmine-imagediff": {},
-        "jasmine-stealth": {
+        "common/js/spec_helpers/jasmine-extensions": {
```

Comment:
@andy-armstrong 

1. `jasmine-extensions.js` and `jasmine-waituntil.js` were written by us.
2. The upstream `jasmine-stealth` is not compatible with Jasmine 2.0. This file contains only one helper from it that is needed in edx-platform.